backend/app/main.py

```diff
@@ -3,6 +3,7 @@
 from app.config import Environment
 from app.config import get_settings
 from app.db import models
+from app.db.cache import redis
```

Comment:
Reminder: The cache will live forever, so we will need a way clear outdated caches

Reply:
Done